<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>propBaseObject</class>
 <widget class="QWidget" name="propBaseObject">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>180</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>180</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>200</width>
    <height>180</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QGroupBox" name="baseObject">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>5</y>
     <width>194</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>BaseObject - Id: 0000000000</string>
   </property>
   <widget class="QCheckBox" name="selected">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>selected</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="visible">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox::indicator {
    image: url(:/icons/HideSelf.ico);
}

QCheckBox::indicator:checked {
    image: url(:/icons/Visible.ico);
}

QCheckBox::indicator:unchecked {
    image: url(:/icons/Hidden.ico);
}

</string>
    </property>
    <property name="text">
     <string>visibility:</string>
    </property>
    <property name="tristate">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Label: </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="labelEdit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="descrEdit">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>91</y>
      <width>180</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>74</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Description:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="visibleKids">
    <property name="geometry">
     <rect>
      <x>165</x>
      <y>18</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox::indicator {
    image: url(:/icons/HideSelf.ico);
}

QCheckBox::indicator:checked {
    image: url(:/icons/Visible.ico);
}

QCheckBox::indicator:unchecked {
    image: url(:/icons/Hidden.ico);
}

</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="dpVision.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>selected</sender>
   <signal>stateChanged(int)</signal>
   <receiver>propBaseObject</receiver>
   <slot>changedSelection(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>59</x>
     <y>58</y>
    </hint>
    <hint type="destinationlabel">
     <x>99</x>
     <y>41</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>visible</sender>
   <signal>stateChanged(int)</signal>
   <receiver>propBaseObject</receiver>
   <slot>changedVisibility(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>129</x>
     <y>58</y>
    </hint>
    <hint type="destinationlabel">
     <x>99</x>
     <y>41</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>labelEdit</sender>
   <signal>textEdited(QString)</signal>
   <receiver>propBaseObject</receiver>
   <slot>changedLabel(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>34</y>
    </hint>
    <hint type="destinationlabel">
     <x>99</x>
     <y>37</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>descrEdit</sender>
   <signal>textChanged()</signal>
   <receiver>propBaseObject</receiver>
   <slot>onDescrChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>101</x>
     <y>131</y>
    </hint>
    <hint type="destinationlabel">
     <x>99</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>visibleKids</sender>
   <signal>toggled(bool)</signal>
   <receiver>propBaseObject</receiver>
   <slot>changedKidsVisibility(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>177</x>
     <y>32</y>
    </hint>
    <hint type="destinationlabel">
     <x>99</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>changedSelection(int)</slot>
  <slot>changedVisibility(int)</slot>
  <slot>changedLabel(QString)</slot>
  <slot>onDescrChanged()</slot>
  <slot>changedKidsVisibility(bool)</slot>
 </slots>
</ui>
